core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -147,20 +149,31 @@ fun NewsResourceCardExpanded(
 fun NewsResourceHeaderImage(
```

Comment:
This Composable is used in a lazy layout, and the [documentation](SubcomposeAsyncImage) for `SubcomposeAsyncImage` advices against using it in lazy layouts because of performance. Could you use something lighter for a loading indicator?

I'll also ask the design team what their UI guidance for what this should look like.

Reply:
The documentation link is broken

Reply:
<img width="733" alt="Screenshot 2023-08-11 at 10 55 42 AM" src="https://github.com/android/nowinandroid/assets/30949634/bfc98080-8cf6-457d-9af3-9653fe8dd646">

here it is @mtali